libcontainer/init_linux.go

```diff
@@ -525,10 +526,10 @@ func setupUser(config *initConfig) error {
 		}
 	}
 
-	if err := system.Setgid(execUser.Gid); err != nil {
+	if err := syscall.Setgid(execUser.Gid); err != nil {
```

Comment:
```suggestion
	if err := unix.Setgid(execUser.Gid); err != nil {
```

Reply:
@AkihiroSuda `unix` implementation just calls `syscall`, so from that perspective `syscall` looks better.